frontend/src/component/changeRequest/ChangeRequest/Changes/Change/ReleasePlanChange.tsx

```diff
@@ -120,6 +127,77 @@ const StartMilestone: FC<{
     );
 };
 
+const ChangeSafeguard: FC<{
+    change: IChangeRequestChangeSafeguard;
+    currentReleasePlan?: IReleasePlan;
+    changeRequestState: ChangeRequestState;
+    environmentName: string;
+    actions?: ReactNode;
+    onSubmit: (data: CreateSafeguardSchema) => void;
+}> = ({
+    change,
+    currentReleasePlan,
+    changeRequestState,
+    environmentName,
+    actions,
+    onSubmit,
+}) => {
+    const releasePlan =
+        (changeRequestState === 'Applied' || !currentReleasePlan) &&
+        change.payload.snapshot
+            ? change.payload.snapshot
+            : currentReleasePlan;
+
+    if (!releasePlan) return;
+
+    const safeguard = change.payload.safeguard;
+
+    if (!safeguard) return;
+
+    const readonly =
+        changeRequestState === 'Applied' || changeRequestState === 'Cancelled';
+
+    return (
+        <StyledTabs>
+            <ChangeItemWrapper>
+                <ChangeItemInfo>
+                    <Added>Change safeguard</Added>
+                    <Typography component='span'>
+                        {safeguard.impactMetric.metricName}
+                    </Typography>
+                </ChangeItemInfo>
+                <div>
+                    <TabList>
+                        <Tab>View change</Tab>
+                        <Tab>View diff</Tab>
+                    </TabList>
+                    {actions}
+                </div>
+            </ChangeItemWrapper>
+            <TabPanel>
+                {readonly ? (
+                    <ReadonlySafeguardDisplay safeguard={safeguard} />
+                ) : (
+                    <SafeguardForm
```

Comment:
before applied CR we can edit it inline using identical form as the original form that created this CR